book/src/getting_started.md

```diff
@@ -25,17 +40,79 @@ Rust compiler. Here are the system requirements (they're pretty modest):
 
 [ru]: https://www.rustup.rs/
 
-There are two ways to get started working with Amethyst:
+## Rust project basics
+
+**If you consider yourself a rust veteran you can probably skip this section,
+just create a binary project and add a dependency for amethyst from crates.io.**
+
+Once you have that setup you'll need to make a new cargo binary project for your
+game to be written in.  You can do this with the following command
+```
+$ cargo init --bin my_cool_game
+```
+Where my_cool_game is the name of your project.
+
+Next you'll want to add a dependency to amethyst.  To do this visit
+[https://crates.io/crates/amethyst] and grab the latest version number from that
+page.  Now go to the Cargo.toml file inside your project directory and add
+```toml
+[dependencies]
+amethyst = "x.x"
+```
+where x.x are the first two sections of the version number you grabbed from
+amethyst.  This way you can receive non-breaking patches from us with
+`cargo update` but you won't receive breaking changes unless you change your
+version number in Cargo.toml.
+This is because amethyst follows [semver](http://semver.org/spec/v2.0.0.html).
```

Comment:
Could we have Amethyst written with an uppercase "A"?

Reply:
Sure